src/JuMP.jl

```diff
@@ -188,8 +188,13 @@ end
 MathProgBase.numvar(m::Model) = m.numCols
 MathProgBase.numlinconstr(m::Model) = length(m.linconstr)
 MathProgBase.numquadconstr(m::Model) = length(m.quadconstr)
+# Note SOC constraints are captured in numquadconstr
```

Comment:
SOC constraints are only captured in ``numquadconstr`` if they're specified as quadratic constraints. If they're specified as SOC constraints using the ``norm`` syntax then they will not be counted there.